rector.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
Did you intentionally  include this file in the commit, @nickygerritsen ?

Reply:
nope